apps/namehashlabs.org/components/2 - molecules/products-section.tsx

```diff
@@ -94,12 +95,13 @@ const Product = ({
                 <a
                   href={buttonUrl}
                   target={withoutExternalLinkIconInCTA ? undefined : "_blank"}
-                  className="border rounded-[8px] bg-black text-white px-4 py-2 transition-colors duration-200 hover:bg-gray-800 inline-flex items-center justify-center"
                 >
-                  {buttonLabel || "Learn more"}
-                  {!withoutExternalLinkIconInCTA && (
-                    <ExternalLinkIcon className="ml-3 w-5 h-5" />
-                  )}
+                  <Button>
+                    {buttonLabel || "Learn more"}
+                    {!withoutExternalLinkIconInCTA && (
+                      <ExternalLinkIcon className="w-5 h-5" />
```

Comment:
@lightwalker-eth I was considering removing the IconButton component. This would give us more flexibility when working with the Button component. What do you think?

Reply:
@eduramme Sorry I don't follow this idea to remove the `IconButton` component.

I understand in theory that there might be some special future case where we have an "Icon Button" in some form that doesn't make sense to use `IconButton` for, but the way our apps and landing pages are currently implemented, it seems to me that pretty much all cases where we have an "Icon Button" should be able to use `IconButton`, right?

Is there some key problem or limitation with `IconButton`? If so, please chat with Jamie about this in our team Slack channel. Thank you :+1:

Reply:
@lightwalker-eth  Sounds good